internal/contracts/oracle_contract/go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/nspcc-dev/neo-go/examples/oracle
+
+go 1.17
+
+require github.com/nspcc-dev/neo-go/pkg/interop v0.0.0-20220713145417-4f184498bc42
```

Comment:
And probably dependency checker is complaining due to this line. We need to update the interop version.

Reply:
Fixed.